### ISSUE TITLE
Protected domains

### ISSUE THREAD

**Issue by sbruggmann** [sandstorm/synco#7]
If the domain is protected, like on stage domains with a basic auth, it does need additional server config, to make it work.

One way could be a static url prefix which can be opened on the server by default, or just something /.well-known/synco-token

**Comment by skurfuerst** [comment]
good point :) I'll try to add to the wizard that it asks for the user and password in this case.

**Comment by jonnitto** [comment]
Hi there, my script hero!

Any progress on this?

**Comment by skurfuerst** [comment]
Hey @jonnitto ,

The workaround for URLs behind HTTP basic authentication is to specify the URL with:

https://user:password@domain.com - we have recently tested this and it works. As soon as I find some time I also want to include this in synco; but it is hard to estimate currently.

All the best,
Sebastian
